db/payment_icons.yml

```diff
@@ -946,3 +946,11 @@
   name: satispay
   label: Satispay
   group: wallets
+-
+  name: archarge
+  label: Archarge
+  group: credit_cards
+-
+  name: qrph
+  label: QR Ph
+  group: other
```

Comment:
Is there a QRPH icon?

Also can you rebase your branch from `master` branch in order to avoid conflicts?

Reply:
Sorry I accidentally got that in mine and then pushed. My team/role usually does not usually work in GH or do this sort of thing, are you able to assist/show how to get this finished, I just don't want to mess it up.  Thanks

Reply:
I think the easiest thing is to re-create the pull request and put the new icon. Could you do that?

Reply:
OK I will close this and try to recreate then.

Reply:
Hey sorry about all this here is new PR: https://github.com/activemerchant/payment_icons/pull/442 I realize I messed up the branching during one of the changes and that sort of threw me off. Thanks again for your help